src/command_modules/azure-cli-iot/azure/cli/command_modules/iot/_help.py

```diff
@@ -15,4 +15,9 @@
 helps['iot device create'] = """
             type: command
             short-summary: Register a device in your Azure IoT Hub.
+"""
+
+helps['iot'] = """
+    type: group
+    short-summary: Connect, monitor, and control millions of IoT assets
```

Comment:
I just stuck with the marketing text; I'll let the IoT team make a change if they would like.